project/Extra.scala

```diff
@@ -8,11 +8,26 @@ import bintray.BintrayKeys.{bintrayRepository, bintrayOrganization}
 
 object Extra {
 
-  val sbtPluginSettings = ScriptedPlugin.scriptedSettings ++ Seq(
+  val sbtPluginSettings = ScriptedPlugin.scriptedSettings.filterNot(
+      _.key.key.label == libraryDependencies.key.label) ++ Seq(
+      // https://github.com/sbt/sbt/issues/3325
```

Comment:
We can't execute scripted tests because `sbt-crossproject-test` and `sbt-scalajs-crossproject` depends on `"org.scala-js" % "sbt-scalajs" % "0.6.16"` which does not support sbt 1.0.0-RC2. 

> Is the workaround implemented here sufficient to get scripted to work with 1.0.0-RC2

You’re almost right. but without this workaround, sbt throw `scala.MatchError` **when just execute `sbt "^^ 1.0.0-RC1"`** not `scripted`.

BTW, `sbt-crossproject` does not have scripted tests yet. Should we remove scripted tests settings from 
`sbt-crossproject`? This workaround unnecessary if remove scripted tests settings from 
`sbt-crossproject`